tests/mixins/stream_hls.py

```diff
@@ -154,10 +144,7 @@ def run(self):
             # at least one read was attempted
             self.read_once.set()
             # only read once per step
-            self.read_wait.wait()
-            self.read_wait.clear()
-
-            try:
+            with self.handshake(OSError) as cm:
```

Comment:
What is `cm` standing for here?

Reply:
- https://github.com/streamlink/streamlink/pull/5017/files#diff-e39ed681d4d3630ba69d1d7b2252e96a1ea51add69bcfd7678d0a7c002984ea9R26-R38
- https://docs.python.org/3/reference/compound_stmts.html#with